### PR TITLE
Site features Pull Request

### PR DESCRIPTION
**Pull request by GloireIrenge**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+body{
```
Comment (GloireIrenge) [inline css/index.css:1]: _Use two spaces instead of tabulation to ensure you respect the CSS best practice_

File: index.html [hunk 2]
```diff
@@ -44,8 +44,6 @@ <h2>Would you recommend freeCodeCamp to a friend?</h2>
         <label for="Maybe">Maybe</label><br>
         <input type="radio" value="Not sure" id="NoSure">
         <label for="NoteSure">Note sure</label>
-
-
         <label for="">Front-end Projects</label>
         <input type="checkbox">
```
Comment (GloireIrenge) [inline index.html:48]: ### Be careful with blank lines and indentation. :warning:

1. Please do not add blank lines, spaces, or indentations without a reason.
2. For readability, kindly add blank lines to separate large or logical code blocks.
3. For readability, kindly add two spaces of indentation. Please do not use the tab key. On some code editors, you can set the tab button to do an X amount of spaces.

**Review by GloireIrenge — COMMENTED** [review]
Hi @GloireIrenge ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights
-  Your codes are working  :heavy_check_mark:
-  Some commits are described :heavy_check_mark:

### Required Changes :repeat:
Please check the comments below for more details.

:key: **Kindly Remember to:** 

- Use Linter in order to recognize and solve CSS and HTML syntax and semantic errors.
- Document your repository in a professional way to make it readable by other persons.
- Respect the CSS and HTML best practice 

Cheers and Happy coding!:grinning:

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @GloireIrenge, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] You should be more polite by using `kindly` and `please` rather than using orders 😉, for example, you said:

- Do not add blank lines, spaces, or indentations without a reason.

But it should be like this:

- Kindly don't add blank lines, spaces, or indentations without a reason.

Kindly change your comment to reflect what I said. 😁

---

- [ ] Also you should give a reason of why you're requesting a change, for example instead of saying:

- Why don't you use an article tag here instead of a div?

You should say:

- Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible.

Please change your comment to reflect what I said. 😁


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Consider using emojis and gifs in your review please, as it makes it look more human-friendly.
- [ ] Please know that you should use the "Start review" feature when reviewing rather than putting single comments each time, to explain, you start by highlighting the piece of code you want to write a comment about then you get this screen: 😁
image
As you can see you have the option to start reviewing and posting a single comment, you should start reviewing as it makes your comments pending and not posted they'll only be posted after you click on this button and put your summary: 😁
image

---

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @GloireIrenge , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️
- Good follow-up with the previous reviewer.

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**